docs/modules/ROOT/pages/general-info/supported-platforms.adoc

```diff
@@ -6,20 +6,24 @@ details the JVMs currently supported by the Payara Platform Community (both Paya
 
 == Supported Java Virtual Machines
 
-The Payara Platform currently supports the following Java Virtual Machines:
+The Payara Platform currently runs the following Java Virtual Machines:
 
-* Oracle JDK8 (u162+), Oracle JDK 11 (11.0.5+)
-* Azul Zulu JDK8 (u162+), Azul Zulu JDK 11 (11.0.5u10+)
-* OpenJDK JDK8 (u162+), OpenJDK 11 (11.0.5+)
-* All other JVMs based on OpenJDK 8u162+ or 11.0.5+ (for example Corretto JDK, Liberica JDK, AdoptOpenJDK)
+* Azul Zulu JDK: 8 (u162+), 11 (11.0.5u10+)
+* Oracle JDK: 8 (u162+), 11 (11.0.5+)
+* Amazon Corretto: 8, 11 (11.0.5+)
```

Comment:
Fro what I can see, you can only download the latest version of 8 and 11 from Coretto Website. So probably best to remove`(11.0.5+)`

Reply:
It's possible to download older ones from here: https://github.com/corretto/corretto-11/releases. The official web site links to this page for archived versions.